jets/d/in_del.c

```diff
@@ -0,0 +1,105 @@
+/* j/4/in_del.c
+**
+*/
+#include "all.h"
+
+/* functions
+*/
+  static u3_noun
+  _rebalance(u3_noun a)
+  {
+    u3_noun l_a, n_a, r_a;
+
+    if ( c3n == u3r_trel(a, &n_a, &l_a, &r_a) ) {
+      return u3m_bail(c3__exit);
+    }
+    else {
+      if ( u3_nul == l_a) {
+        return u3k(r_a);
```

Comment:
Don't we need to `u3z(n_a)` if we're returning just `l_a` or `r_a`?

This might not show up as a memory error if you've only tested this with sets whose items are direct atoms.

Reply:
Eh, I'll reread this assuming RETAIN semantics, instead of TRANSFER. I wish jets were more explicit about that.